external-import/ransomwarelive/src/lib/ransomConn.py

```diff
@@ -463,11 +472,18 @@ def stix_object_generator(self, item, group_data):
                 external_references.append(external_reference)
 
         # Creating Report object
+        report_name = (
+            item.get("group_name")
+            + " has published a new victim: "
+            + item.get("post_title")
```

Comment:
Note : this suggestion might be irrelevant if we do not want emoticon in the report title.

```suggestion
            + victim_name
```